--- /etc/skel/.bashrc	2024-03-31 16:41:03.000000000 +0800
+++ .bashrc	2024-11-09 17:58:22.756876947 +0800
@@ -57,9 +57,9 @@
 fi
 
 if [ "$color_prompt" = yes ]; then
-    PS1='${debian_chroot:+($debian_chroot)}\[\033[01;32m\]\u@\h\[\033[00m\]:\[\033[01;34m\]\w\[\033[00m\]\$ '
+    PS1='${debian_chroot:+($debian_chroot)}\[\033[01;32m\]\u@\h\[\033[00m\]:\[\033[01;34m\]\W\[\033[00m\]$(__git_ps1 ":\[\033[01;33m\]%s\[\033[00m\]")\$ '
 else
-    PS1='${debian_chroot:+($debian_chroot)}\u@\h:\w\$ '
+    PS1='${debian_chroot:+($debian_chroot)}\u@\h:\W$(__git_ps1 ":%s")\$ '
 fi
 unset color_prompt force_color_prompt
 
@@ -115,3 +115,18 @@
     . /etc/bash_completion
   fi
 fi
+
+# Set up less.
+export LESS=-FRSX
+
+rc_dir="${HOME}/.bashrc.d"
+if [[ -d "${rc_dir}" && -r "${rc_dir}" && -x "${rc_dir}" ]]; then
+    for i in "${rc_dir}"/*.sh; do
+        [[ -f "${i}" && -r "${i}" ]] && . "${i}"
+    done
+fi
+unset rc_dir i
+
+#
+# NOTE: Changes below should be moved to ~/.bashrc.d.
+#
